### PR TITLE
Infrastructure - README

### PR DESCRIPTION
**Pull request by Thurman1776-**
This attempts to resolve - https://github.com/Thurman1776-/Uncomplect-us/issues/1

### CODE REVIEW

**Comment by Thurman1776-** [comment]
Hey @emrepun 

I'm lacking a bit of ideas here for the README 😅 
So I'm requesting your review to hear any good suggestions from you! 🙇

File: Mac-App/Mac-App/Info.plist [hunk 1]
```diff
@@ -20,6 +20,8 @@
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>LSApplicationCategoryType</key>
+	<string>public.app-category.developer-tools</string>
```
Comment (Thurman1776-) [inline Mac-App/Mac-App/Info.plist:24]: Infiltrating unrelated change 🕵️‍♂️

**Comment by emrepun** [comment]
Yey! Soon we are going to have a README 🎉 Sorry I missed this due to vacation, will review it soon @Thurman1776-

**Comment by emrepun** [comment]
> Hey @emrepun
> 
> I'm lacking a bit of ideas here for the README 😅
> So I'm requesting your review to hear any good suggestions from you! 🙇

Since you mention the usage of SwiftUI, maybe mentioning ReSwift would also be helpful? @Thurman1776-

**Comment by emrepun** [comment]
Having a screenshot with results of a sample project might also be useful, maybe can also have a screenshot of a class to show how it works (I know UI might change in the future but for the time being it would be nice) @Thurman1776-

**Review by emrepun — APPROVED** [review]
🎉

**Comment by Thurman1776-** [comment]
@emrepun The images aren't loading & I assume it is because I'm using the diff url instead of a resource url. 
I'd need to update them after this is merged. Unless you know if it's possible to do it in a different way?

**Comment by emrepun** [comment]
> @emrepun The images aren't loading & I assume it is because I'm using the diff url instead of a resource url.
> I'd need to update them after this is merged. Unless you know if it's possible to do it in a different way?

Hmm, Im not sure why it might be happening. Lets merge this one then! @Thurman1776-
